src/oph/ehoks/palaute/scheduler.clj

```diff
@@ -25,7 +26,12 @@
   initiate all palautteet for those HOKSes."
   [opts]
   (log/info "Initialising palautteet for next batch of uninitialised HOKSes.")
-  (palaute/reinit-palautteet-for-uninitiated-hokses! 3000))
+  (let [result (util/with-timeout
+                 #(palaute/reinit-palautteet-for-uninitiated-hokses! 3000)
```

Comment:
Eiks täs `f` ja `timeout-millis` oo väärin päin?

Reply:
joo, sorry :)

Reply:
kiitos tarkkuudesta <3